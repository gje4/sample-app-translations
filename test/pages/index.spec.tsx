import Index from '@pages/index';
import { render, screen } from '@test/utils';

jest.mock('@lib/hooks', () => require('@mocks/hooks'));

describe('Homepage', () => {
    test('renders correctly', () => {
        const { container } = render(<Index />);
        const heading = screen.getByRole('heading', { level: 2 });

        expect(container.firstChild).toMatchSnapshot();
        expect(heading).toBeInTheDocument();
    });
});
