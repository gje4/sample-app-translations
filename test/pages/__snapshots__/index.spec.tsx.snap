// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders correctly 1`] = `
<div
  class="styled__StyledBox-sj5f1m-0 htmbUi styled__StyledPanel-sc-1h6ef3q-0 jooDHr"
>
  <div
    class="styled__StyledBox-sj5f1m-0 jvmRZj styled__StyledFlex-hcvk8l-0 kObzyk"
  >
    <h2
      class="styled__StyledH2-tqnj75-2 styled__StyledH2-sc-1h6ef3q-1 fJTAEy"
    >
      Homepage
    </h2>
  </div>
  <div
    class="styled__StyledBox-sj5f1m-0 jvmRZj styled__StyledFlex-hcvk8l-0 cyJYsQ"
  >
    <div
      class="styled__StyledBox-sj5f1m-0 cqnBsC pages__StyledBox-jwwna7-0 hLZLBD"
    >
      <h4
        class="styled__StyledH4-tqnj75-4 clKqyF"
      >
        Inventory count
      </h4>
      <h1
        class="styled__StyledH1-tqnj75-1 kOjopT"
      >
        3
      </h1>
    </div>
    <div
      class="styled__StyledBox-sj5f1m-0 cqnBsC pages__StyledBox-jwwna7-0 hLZLBD"
    >
      <h4
        class="styled__StyledH4-tqnj75-4 clKqyF"
      >
        Variant count
      </h4>
      <h1
        class="styled__StyledH1-tqnj75-1 kOjopT"
      >
        2
      </h1>
    </div>
    <div
      class="styled__StyledBox-sj5f1m-0 goqkha pages__StyledBox-jwwna7-0 hLZLBD"
    >
      <h4
        class="styled__StyledH4-tqnj75-4 clKqyF"
      >
        Primary category
      </h4>
      <h1
        class="styled__StyledH1-tqnj75-1 kOjopT"
      >
        widgets
      </h1>
    </div>
  </div>
</div>
`;
